### DISCUSSION TITLE
Visit URL during exam

### DISCUSSION THREAD

**Discussion by cpoloal** [SafeExamBrowser/seb-win-refactoring#418] [Q&A]
Hi,

we are testing SEB for Windows and Mac in Moodle. Our requirements include to assure students can visit external URLs while completing an exam. We tried to do it configuring the settings included in Network tab. As a result, we can visit certain URLs, but we are not able to come back to Moodle in order to issue the answers there.

Is there anything else we need to check?

Thanks

**Comment by danschlet** [comment]
You would need to make sure these external sites open in new browser windows. I'm not sure how Moodle handles that, in plain HTML you need to use the parameter `target = _blank`. Also SEB has the feature called ["Back to Start" button](https://safeexambrowser.org/macosx/mac_usermanual_en.html#ExamPane), when enabled students can click that and it will bring them back to the start URL of a quiz or a custom URL.
